DoCoMoChatReq request
---
bool success
DoCoMoChatRes response 
